.github/workflows/cypress-tests.yml

```
@@ -6,8 +6,6 @@ name: Cypress Tests
 on:
   push:
     branches: [ "main" ]
```

```suggestion
    branches: [ "main", "cypress-tests" ]
```

To check if it works on this current branch